testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -245,10 +245,8 @@ fn block_proposal_rejection() {
     );
 
     info!("------------------------- Test Block Proposal Rejected -------------------------");
-    // Verify that the node correctly rejected the node
-    let proposed_signer_signature_hash =
-        signer_test.wait_for_validate_reject_response(short_timeout);
-    assert_eq!(proposed_signer_signature_hash, block_signer_signature_hash);
+    // Give signer time to reject block
+    std::thread::sleep(Duration::from_secs(3));
```

Comment:
I would actually do the below code on a loop with waits and timeout similar to other parts of the various tests as knowing CI, this may cause issues XD

Reply:
Agree, added a polling loop